src/site/README.md

```diff
@@ -315,13 +315,158 @@ The Liquid Template Unit Testing Framework was created to replace Cypress for un
 **Please note**: End-to-end (e2e) tests on VA.gov use Cypress; Cypress has NOT been replaced by this tool.
 
 ### How to Use the Framework
```

Comment:
Can we just link over to the real docs for this instead of duplicating it? Some concerns about the two falling out of sync

Reply:
I totally agree. I replaced the content of this section with a link to the doc in the `va.gov-team` repo.